deleteTempCache.php

```diff
@@ -0,0 +1,11 @@
+// This scipt is called by an administrator of the Formulize install within the Admin section of the site, in order to remove all files in the 'templates_c' folder.
```

Comment:
Comments need to be after the <?php tag because otherwise they'll be sent to the browser, as any text would.